docs/contributing/style-guide.md

```diff
@@ -106,7 +150,9 @@ For example, the first time you use AWS Application Load Balancer (ALB), you int
 
 ## Beta Releases
 
-For a Closed Beta release, you'll need to exclude it from the nav and search engine results so that users will need the link to access it:
+### Closed Beta
```

Comment:
Can you add a sentence that describes what a closed beta is?

Reply:
I'll create an Asana ticket for defining open/closed beta.